### DISCUSSION TITLE
Install and run without admin permissions

### DISCUSSION THREAD

**Discussion by ponchofiesta** [JanDeDobbeleer/oh-my-posh#2394] [Troubleshoot]
Is there a way to install and run oh-my-posh without admin permissions?

**Comment by JanDeDobbeleer** [comment]
What do you mean? The default installation method is user no?

**Reply by ponchofiesta** [reply]
I got a message telling unsigned script execution is blocked. I assume this has to be turned of by an administrator, correct?

**Reply by JanDeDobbeleer** [reply]
@ponchofiesta where do you get this message? Can I get a screenshot?

**Reply by ponchofiesta** [reply]
I followed the install docs: https://ohmyposh.dev/docs/installation/prompt
And `. $PROFILE` doesn't work:
```
PS C:\Users\XXX> . $PROFILE
. : Die Datei "C:\Users\XXX\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1" kann nicht geladen
werden, da die Ausführung von Skripts auf diesem System deaktiviert ist. Weitere Informationen finden Sie unter
"about_Execution_Policies" (https:/go.microsoft.com/fwlink/?LinkID=135170).
In Zeile:1 Zeichen:3
+ . $PROFILE
+   ~~~~~~~~
    + CategoryInfo          : Sicherheitsfehler: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
```

Same error message occurs if I restart Powershell.

**Reply by ponchofiesta** [reply]
OK, got it working by doing `Set-ExecutionPolicy Bypass -Scope CurrentUser`

**Reply by ponchofiesta** [reply]
Font installation didn't work too:
```
oh-my-posh font install

    open C:\WINDOWS/Fonts/Meslo LG S Italic Nerd Font Complete Mono Windows Compatible.ttf: Access is denied.

``` 

Fix: Download manually and do right click -> Install (installs in user context instead of C:\Windows\Fonts)

**Reply by JanDeDobbeleer** [reply]
@ponchofiesta font installation is beta, it's still not functional on Windows
